hw/top_earlgrey/data/ip/chip_sram_ctrl_testplan.hjson

```diff
@@ -139,7 +134,8 @@
              - Read the location from the SRAM and check that the data has been wiped.
              - Repeat the steps above for the retention SRAM.
 
-             '''
+             
+            '''
```

Comment:
It's nice it spotted the incorrect indenting, but a shame it has to keep the same semantics so it has to add a newline